.kokoro/presubmit/presubmit.cfg

```diff
@@ -5,7 +5,3 @@ env_vars: {
     key: "RUN_SYSTEM_TESTS"
     value: "false"
 }
-env_vars: {
```

Comment:
@busunkim96 How should we resolve this?

I assume we'll need to change `synth.py` to not update `presubmit.cfg` until https://github.com/googleapis/python-bigquery/pull/203 can be merged and we can remove this logic?

Reply:
@tswast Yup, I think adding it to the `excludes` list would be easiest.